maven-compat/pom.xml

```diff
@@ -100,6 +100,10 @@ under the License.
       <groupId>org.apache.maven.wagon</groupId>
       <artifactId>wagon-provider-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+    </dependency>
```

Comment:
`commons-io` is not included in the distribution, I'd rather not add it.

Reply:
I agree here because I moved to to test scope because we just don't need it.

Reply:
It's probably in the classpath anyway. Maybe next decade we can move to Java 11 and get rid of this too. :-)

Reply:
Yeah, I really thought you were kidding here... 😭